tools/biobb_pytorch/biobb_apply_mdae.xml

```diff
@@ -0,0 +1,103 @@
+<tool id="biobb_pytorch_apply_mdae" name="ApplyMdae" version="@TOOL_VERSION@" profile="22.05">
+    <description>Apply a Molecular Dynamics AutoEncoder (MDAE) PyTorch model.</description>
+    <macros>
+      <token name="@TOOL_VERSION@">4.2.1</token>
+    </macros>
+
+    <requirements>
+      <requirement type="package" version="@TOOL_VERSION@">biobb_pytorch</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+
+      ln -s '$input_data_npy_path' ./input_data_npy_path.$input_data_npy_path.ext &&
+      ln -s '$input_model_pth_path' ./input_model_pth_path.$input_model_pth_path.ext &&
+      #if $config_json:
+        ln -s '$config_json' ./config_json.$config_json.ext &&
+      #end if
+
+      apply_mdae
+
+      #if $config_json:
+        --config ./config_json.$config_json.ext
+      #end if
+
+      --input_data_npy_path ./input_data_npy_path.$input_data_npy_path.ext
+      --input_model_pth_path ./input_model_pth_path.$input_model_pth_path.ext
+      --output_reconstructed_data_npy_path $outname_output_reconstructed_data_npy_path
+      #if $outname_output_latent_space_npy_path:
+        --output_latent_space_npy_path $outname_output_latent_space_npy_path
+      #end if
+      ;
+
+      mv '$outname_output_reconstructed_data_npy_path' '$output_reconstructed_data_npy_path' &&
+      if test -f '$outname_output_latent_space_npy_path'; then mv '$outname_output_latent_space_npy_path' '$output_latent_space_npy_path'; fi;
+
+      ]]>
+    </command>
+
+    <inputs>
+      <param name="input_data_npy_path"  type="data" format="npy"  optional="False" label="Input NPY file" help="Input data file"/>
+      <param name="input_model_pth_path" type="data" format="pth"  optional="False" label="input PTH file" help="Path to the input model file. Format: [input].pth"/>
```

Comment:
What is this? I thought this is a second input file, but the description indicates its a path to something?

Reply:
Fixed, context errors of using an automatic generator.t